source/reference/glossary.txt

```diff
@@ -585,6 +585,18 @@ Glossary
       basic mechanism enabling :term:`replication` in MongoDB.
       See :doc:`/core/replica-set-oplog`.
 
+   optime
+      .. versionchanged:: 3.2
+
+         MongoDB 3.2 introduced replication
+         :rsconf:`protocolVersion: 1 <protocolVersion>` with a new optime
+         format.
```

Comment:
I see. How about

    The following describes the optime format used by `protocolVersion: 1`, introduced by MongoDB 3.2.